### DISCUSSION TITLE
Bug in multilayer search

### DISCUSSION THREAD

**Discussion by tapiero** [tomickigrzegorz/leaflet-examples#47] [General]
Hi, 
I was implementing multilayersearch in my app, and I noticed a bug: doing the search for points, and selecting the result, I noticed that the map zoom brings you back to wrong coordinates, more precisely if I decrease the zoom level and do a search, the map does not rest me at the selected point and does not give me the right zoom level. This bug is not part of my app, because I noticed it also happens on your site at the link: https://tomik23.github.io/leaflet-examples/#45.multi-layer-search

I'm sorry it doesn't work, and I hope it fixes the bug, anyway thanks for the plugin

**Comment by tomickigrzegorz** [comment]
Yes, I know these examples are not free from errors ;)
My intention to create such a website was not to give complete solutions that can be used by copy/paste to your project. The examples are designed to give direction to those who learn leaflejs.
I know of a few bugs in this as well. I leave these bugs for people who take this code to fix the problems themselves, because I believe that this is learning about finding solutions.

So if you know how and you want to correct this error, you can put it here as a hint for others who have less experience than you ;)
I would like this page and examples to "live", unfortunately there is little response to #38  a pity.

Thanks for bringing up this topic.

Good luck in your studies.
